language/README.md

```diff
@@ -92,9 +96,9 @@ Examples:
   node analyze.v1beta2.js entities-file my-bucket file.txt      Detects entities in gs://my-bucket/file.txt
   node analyze.v1beta2.js syntax-text "President Obama is speaking at the White House."
   node analyze.v1beta2.js syntax-file my-bucket file.txt        Detects syntax in gs://my-bucket/file.txt
-  node analyze.v1beta2.js entity-sentiment-text "President Obama is speaking at the White House."
-  node analyze.v1beta2.js entity-sentiment-file my-bucket       Detects sentiment of entities in gs://my-bucket/file.txt
-  file.txt
+  node analyze.v1beta2.js classify-text "Android is a mobile operating system developed by Google."
```

Comment:
This text is too short - currently the API requires 20 tokens to return non-empty results.  This limitation might be improved in the future but let's use a longer piece of text for the sample just in case.